pom.xml

```diff
@@ -72,6 +72,12 @@
                 <version>2.11.0</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.tika</groupId>
+                <artifactId>tika-core</artifactId>
+                <version>1.28.3</version>
```

Comment:
Please update the version we are using instead of adding a new dependency entry. [It is in this same file, line 290](https://github.com/ONSdigital/zebedee/blob/c221b7db376d8c6189224328f2a1da3be1a5350f/pom.xml#L290)